packages/docs/pages/experimental/logging-config.mdx

```diff
@@ -0,0 +1,17 @@
+# Configuring Logging on Namada
+
+##### `NAMADA_LOG`
```

Comment:
```suggestion
## `NAMADA_LOG`
```

Reply:
For prettiness reasons, I prefer to use #### , since Nextra renders these too large otherwise. I hope this is ok.